### PR TITLE
Add missing handling for `ActionState::consumed`

### PR DESCRIPTION
**Pull request by Shute052**
Missing handling with `ActionState::consumed` was added back, which was overlooked in #450.

image

To be specific, `ActionState::press(Action)` takes `consumed` into account, whereas `ActionData::state::press(Action)` did not.

### CODE REVIEW

**Comment by Shute052** [comment]
Both 0.12.2 and 0.13.1?

File: RELEASES.md [hunk 1]
```diff
@@ -27,6 +27,12 @@
 - `bevy` dependency has been bumped from 0.12 to 0.13.
 - `bevy_egui` dependency has been bumped from 0.24 to 0.25.
 
+## Version 0.12.2
```
Comment (alice-i-cecile) [inline RELEASES.md:30]: This should probably be `0.13.1`. I don't think this is impactful enough to backport without others asking for it.

**Review by alice-i-cecile — CHANGES_REQUESTED** [review]
Change itself looks good, but let's aim for 0.13.1 instead.

**Comment by Shute052** [comment]
Also bump the version number

**Review by alice-i-cecile — APPROVED** [review]

**Comment by Shute052** [comment]
This fixed #443

**Pull request merged**